src/app/Dashboard/AddCard.tsx

```diff
@@ -0,0 +1,145 @@
+ * The above copyright notice and either this complete permission notice or at
+ * a minimum a reference to the UPL must be included in all copies or
+ * substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+import * as React from 'react';
+import {
+  Bullseye,
+  Button,
+  Card,
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateIcon,
+  EmptyStateVariant,
+  Select,
+  SelectOption,
+  Stack,
+  StackItem,
+  Text,
+  Title,
+} from '@patternfly/react-core';
+import { Wizard, WizardStep } from '@patternfly/react-core/dist/js/next';
+import { PlusCircleIcon } from '@patternfly/react-icons';
+import { useDispatch } from 'react-redux';
+import { StateDispatch } from '@app/Shared/Redux/ReduxStore';
+import { addCardIntent } from '@app/Shared/Redux/DashboardConfigActions';
+import { DashboardCards, getConfigByName, getConfigByTitle } from './Dashboard';
+
+interface AddCardProps {}
+
+export const AddCard: React.FunctionComponent<AddCardProps> = (props: AddCardProps) => {
+  const [showWizard, setShowWizard] = React.useState(false);
+  const [selection, setSelection] = React.useState(DashboardCards[0].title);
+  const [selectOpen, setSelectOpen] = React.useState(false);
+  const dispatch = useDispatch<StateDispatch>();
+
+  const options = React.useMemo(() => {
+    return [
+      ...DashboardCards.map((choice, idx) => (
+        <SelectOption key={idx} value={choice.component.name}>
+          {choice.title}
+        </SelectOption>
+      )),
+    ];
+  }, [DashboardCards]);
+
+  const handleToggle = React.useCallback(
+    (isOpen) => {
+      setSelectOpen(isOpen);
+    },
+    [setSelectOpen]
+  );
+
+  const handleSelect = React.useCallback(
+    (_, selection) => {
+      setSelection(selection);
+      setSelectOpen(false);
+    },
+    [setSelection, setSelectOpen]
+  );
+
+  const handleAdd = React.useCallback(() => {
+    setShowWizard(false);
+    dispatch(addCardIntent(getConfigByTitle(selection).component.name));
+  }, [dispatch, selection]);
+
+  const handleStart = React.useCallback(() => {
+    setShowWizard(true);
+  }, [setShowWizard]);
+
+  const handleStop = React.useCallback(() => {
+    setShowWizard(false);
+  }, [setShowWizard]);
+
+  return (
+    <>
+      <Card isRounded isLarge>
+        {showWizard ? (
+          <Wizard onClose={handleStop} onSave={handleAdd}>
+            <WizardStep id="card-type-select" name="Card Type" footer={{ nextButtonText: 'Finish' }}>
+              <Stack hasGutter>
```

Comment:
Maybe we can use the footer prop on `SelectOption` to achieve this. Just looks a little nicer and more compact. Also, the user can see the description without selecting one.

![Screenshot from 2022-12-09 15-04-39](https://user-images.githubusercontent.com/68053619/206787389-e0c8e23d-fd37-4793-b219-d32d183896d8.png)